test/efficiency/test_activations_time.py

```diff
@@ -7,7 +7,7 @@
 @pytest.mark.parametrize("activation", ["tanh", "sigmoid"])
 @assert_time(max_time=10)
 def test_activation(activation, hook, workers):
-
+    "Testing the activation function (sigmoid or tanh) works as expected"
```

Comment:
This isn't triply quoted `"""` as the others are.

Reply:
Changed it. Thanks